Products/ZenUI3/browser/resources/js/zenoss/form/graphPanel.js

```diff
@@ -197,6 +197,7 @@
                                          '     </div> ' +
                                          '    <img id="{buttonId}" class="europaGraphGear" src="/++resource++zenui/img/gear.png"  />' +
                                          '</div>'),
+        maxLinkLength: 4096,
```

Comment:
please add the appropriate comment to describe the reason of maxLinkLength value

Reply:
added